files/m8tro-bootstrap/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "m8tro-bootstrap",
+  "repo": "idleberg/m8tro-bootstrap",
+  "files": {
+    "include": ["./dist/m8tro.css", "./dist/m8tro.min.css"]
```

Comment:
Thats not correct, your settings will create a folder called "dist".
Do this instead:
```
"files": {
    "basePath": "dist/",
    "include": ["m8tro.min.css"]
...
```

Also do you really need the un-minified file? Maybe its best to only hos the min.css?